src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -79,8 +99,21 @@ public static int Main(string[] args)
             app.Description = "Initializes empty project for .NET Platform";
             app.HelpOption("-h|--help");
 
+            var csharp = app.Option("-c#|--csharp", "Create an C# project", CommandOptionType.NoValue);
```

Comment:
`dotnet new fsharp/csharp` seems fine to me. Given most users of the toolchain will be using csharp, it seems like a reasonable default. In 80% of cases, the only thing you'd need to type is `dotnet new`